CommonTools/MVAUtils/interface/GBRForestToolsTinyXML.h

```diff
@@ -0,0 +1,27 @@
+#ifndef CommonToolsTinyXMLs_MVAUtils_GBRForestToolsTinyXML_h
+#define CommonToolsTinyXMLs_MVAUtils_GBRForestToolsTinyXML_h
```

Comment:
The header include guard is badly formatted and does not match the file location.
There is actually no need to modify it compared to the master version (no need to add TinyXML in the name because there is no symbol conflict).

My earlier comment suggesting to add ```TinyXML``` referred only to the types/functions that were in conflict with the already present code (like TMVAEvaluator or methods from TMVAZipReader that were present in the previous version of this PR).